### DISCUSSION TITLE
Noob here. Where do I put my share folder in windows?

### DISCUSSION THREAD

**Discussion by albeezar** [marcone/teslausb#830] [Q&A] [answered]
Hi. I don't understand where on my windows PC do I put my TeslaCam shared folders. And then how do I properly tell my teslausb the exact location of the file? 

So right now I just put the TeslaMusic and TeslaCam folders in C:// directory. 

In export Share name in the configuration file, what is the exact location that I put in? 

Is it just TeslaUsb (the name of my folder on the C drive? 

Or do I have to put 192.168...../Teslausb? 

Do I have to put '' quotes for this location? Or not needed? 

I'm just very lost on the share name location and what to type in the export SHARE_NAME=
For both the Tesla music folder and the TeslaCam folder. 

Thank you for any basic help you can offer!

**Comment by marcone** [comment, marked as answer]
> So right now I just put the TeslaMusic and TeslaCam folders in C:// directory.
> ... 
> Or do I have to put 192.168...../Teslausb?

If you named the folders "TeslaMusic" and "TeslaCam", then obviously you shouldn't use "Teslausb" in the config file.

Here's roughly how it should work when using a Windows PC as the archive server: (this may vary based on Windows version, what I describe is for my Windows 10 PC)
- create a folder on your PC. For this example let's say C:\TeslaVideos
- right-click on the folder and select "Properties"
- in the new window that appears, go to the "Sharing" tab
- click the "Share..." button
- In the "Choose people to share with" dialog that pops up, select a user, or choose "Everyone", then click the "Share" button. 
- The dialog should change to say "Your folder is shared". Click the "Done" button
- Back in the "Sharing" tab, it should now show the network path, probably something like "\\\\DESKTOP-XY1ZF99\TeslaVideos"
- In the TeslaUSB config file, you would put:
```
   export ARCHIVE_SERVER="DESKTOP-XY1ZF99"
   export SHARE_NAME="TeslaVideos"
   export SHARE_USER=_YOUR_WINDOWS_USERNAME_HERE_
   export SHARE_PASSWORD=_YOUR_WINDOWS_PASSWORD_HERE_
```
Fill in the correct username and password. Note that if you're using a Microsoft account instead of a local account, you need to use the actual Microsoft username (i.e. what you'd use to log in to the Microsoft website, not the name you see when you to log in to the PC), and the actual password, not the PIN you may have set up to log in to the PC.
